collect_app/src/main/java/org/odk/collect/android/widgets/DateWidget.java

```diff
@@ -223,7 +223,7 @@ private int getTheme() {
         int theme = 0;
         // https://github.com/opendatakit/collect/issues/1424
         // https://github.com/opendatakit/collect/issues/1367
-        if (!isBrokenSamsungDevice() && Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        if (!isBrokenSamsungDevice()) {
```

Comment:
There is something to improve in that method as well. Please take a look.

Reply:
I mean you can get rid of `&& Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP`

Reply:
Please also get rid of those two comments with links to issues. We can find related issues using git.

Reply:
Still `isBrokenSamsungDevice()` is not updated.

Reply:
What about this @SaumiaSinghal?

Reply:
I read your comment and removed the link to those issues, but jumped `&& Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP`, I updated it.